addons/account/models/chart_template.py

```diff
@@ -160,12 +160,12 @@ def _notify_accountant_managers(taxes_to_check):
 
     env = api.Environment(cr, SUPERUSER_ID, {})
     chart_template_id = env['ir.model.data'].xmlid_to_res_id(chart_template_xmlid)
-    companies = env['res.company'].search([('chart_template_id', '=', chart_template_id)])
+    companies = env['res.company'].search(['|', ('chart_template_id', '=', chart_template_id), ('chart_template_id.parent_id', '=', chart_template_id)])
```

Comment:
@dylankiss I am confused here.  Why do we pass common as the template and then suppose only one child level?  Maybe we could use the child_of param here (or just in a for to calculate all the children of the chart_template_id) and if we pass common, we only update the tax templates of the common template.  (so we could just call with another subtemplate if we change something there)

Reply:
@jco-odoo You're right on both remarks. I changed it to use the `child_of` operator now and only search for tax templates in the given chart.